frontend/src/pages/NewDoctor.js

```diff
@@ -55,13 +76,18 @@ export default function NewDoctor() {
 				/>
 
 				<TextField
-					variant='outlined'
+					id='select-specialty'
+					select
 					value={newDoctor.specialty}
-					placeholder='Specialty'
+					label='Specialty'
 					required={true}
 					name='specialty'
 					onChange={handleChange}
-				/>
+				>
+					{specialties.map((specialty) => (
+						<MenuItem value={specialty.value}>{specialty.value}</MenuItem>
+					))}
```

Comment:
```suggestion
				{specialties.map(({...value}) => (
						<MenuItem key={value} value={value}>{value}</MenuItem>
					))}	
```

Reply:
What is the ellipsis for ('...value')? There's nothing to spread here

Reply:
value is the only thing you need here, saves a lot of typing and makes the code more readable if you spread it instead of retyping specialty.value each time.
You could do the same with your props by the way. i.e. function NewDoctor({...firstName, ...specialty, etc.}) and the use the respective variables instead of retyping NewDoctor.firstName/specialty/whatever each time

Reply:
I got that, I just meant that there is no need for the three dots here. Just {value} is enough